README.md

```diff
@@ -117,6 +117,17 @@ required if you want to run tests. Whether you need other packages will
 depend on your use case.
 
 To run executables or tests, you will need to install PostgreSQL.
+Depending on your setup, you may need to cast some authorization
+spells.  Here is what works on debian (run as user postgres):
+
+```shell
+[as user postgres] $ createuser thentos -d
```

Comment:
to be fully explicit, maybe remote the `as user` stuff and say instead:

`sudo -u postgres createuser ...`